packages/extension/src/components/DaDndMessage.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="dnd-message-bar">
+    <p class="do-not-disturb">Do not disturb.</p>
+    <a href="#" class="turn-off" role="button" @click="$emit('dndOff')">Turn Off</a>
+  </div>
+</template>
+
+<style scoped>
+.dnd-message-bar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 32px;
+  background-color: #7e48ff;
```

Comment:
We have the color palette defined here: https://github.com/dailynowco/daily-apps/blob/master/packages/components/src/styles/colors.pcss
So you don't need to hardcode set the color.
```suggestion
  background: var(--color-onion-40);
```